lib/runner/extract-runnable-items.js

```diff
@@ -1,4 +1,5 @@
 var sdk = require('postman-collection'),
+    _ = require('lodash'),
```

Comment:
I do not see any reason why we should start using lodash here. 
Are we saying we can't use `Array.isArray` or `Array.prototype.includes` cannot be used here? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

Reply:
@saswatds
It was an ESLint suggestion.
So I used it.